README.md

```diff
@@ -73,38 +80,46 @@ nvidia-dra-driver   nvidia-dra-plugin-lt7qh                    1/1     Running
 ### Run the examples by following the steps in the demo script
 Finally, you can run the various examples contained in the `demo/specs/quickstart` folder.
 The `README` in that directory shows the full script of the demo you can walk through.
-```console
+
+```bash
 cat demo/specs/quickstart/README.md
-...
 ```
 
-Where the running the first three examples should produce output similar to the following:
+Depending on the GPU resources in the cluster, running the first three examples will produce output similar to the following:
 ```console
-$ kubectl apply --filename=demo/specs/quickstart/gpu-test{1,2,3}.yaml
-...
-
+kubectl apply --filename=demo/specs/quickstart/gpu-test{1,2,3}.yaml
 ```
+
 ```console
-$ kubectl get pod -A
+kubectl get pod -A -l app=pod
+```
+```
 NAMESPACE           NAME                                       READY   STATUS    RESTARTS   AGE
 gpu-test1           pod1                                       1/1     Running   0          34s
 gpu-test1           pod2                                       1/1     Running   0          34s
 gpu-test2           pod                                        2/2     Running   0          34s
 gpu-test3           pod1                                       1/1     Running   0          34s
 gpu-test3           pod2                                       1/1     Running   0          34s
-...
-
 ```
 ```console
-$ kubectl logs -n gpu-test1 -l app=pod
+kubectl logs -n gpu-test1 -l app=pod
+```
+```
```

Comment:
I looked at the generated README with this change, and I prefer the look of the original. Is the purpose here to make it easier to copy-and-paste the command where the output is attached?

Reply:
Yes, otherwise, a user has to manually delete the character `$`. Also some examples contain the output of a command in the console quote, which makes the command invalid.